### PR TITLE
Update translation features and composer dependencies

### PR DESCRIPTION
**Pull request by JarJak**
See https://github.com/bolt/four/pull/391#issuecomment-482815115

Closes #391

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "description": "Bolt 4 skeleton / prototype",
     "require": {
-        "php": ">=7.1.3",
+        "php": ">=7.2",
```
Comment (bobdenotter) [inline composer.json]: SF 4 uses 7.1.3. We should stick with that. https://symfony.com/doc/current/reference/requirements.html

File: src/Menu/BackendMenuBuilder.php [hunk 2]
```diff
@@ -199,14 +199,6 @@ private function createAdminMenu(): ItemInterface
             ],
         ]);
 
-        $menu->getChild('Maintenance')->addChild('Fixtures', [
```
Comment (bobdenotter) [inline src/Menu/BackendMenuBuilder.php]: Why remove this one? It doesn't work yet, but we'll add it sooner or later.

File: composer.json (hunk 1 above)
Comment (JarJak) [reply composer.json]: PHPUnit 8 requires PHP 7.2 and 7.1 is close to EOL. But I can revert it, no problem.

File: src/Menu/BackendMenuBuilder.php (hunk 2 above)
Comment (JarJak) [reply src/Menu/BackendMenuBuilder.php]: I don't think we could use migrations in Bolt.

Comment (JarJak) [reply src/Menu/BackendMenuBuilder.php]: reverted

File: composer.json (hunk 1 above)
Comment (JarJak) [reply composer.json]: reverted

Comment (bobdenotter) [reply composer.json]: Thanks. I think it's beneficial to us to stick with SF's version requirements :-)

**Pull request merged**